docs/management/direct-k8s-access.md

```diff
@@ -0,0 +1,25 @@
+# Direct Kubernetes Access
+
+Convox makes managing and operating your Kubernetes cluster extremely easy and simple by abstracting a lot of the unnecessary complexities away  We know, however, that there may be times that you will want to dive deeper and access the k8s primitives and resources yourself.
+
+This could be deep debugging of an edge-case networking issue, examination of an underlying cloud infrastructure issue, gaining a deeper understanding of how everything ties together, or simply to install a Kubernetes native component or plugin directly.
+
+Convox provides an API proxy to Kubernetes that runs on your Rack which allows you to grant access to the underlying system, whilst still respecting and adhering to the Role-Based Access Control (RBAC) that Kubernetes provides.  This is actually a lot easier and more manageable than providing direct Kubernetes credentials to the developers in your team!
+
+## Configure kubectl to Point at Your Rack
+
+Convox allows you to securely connect your `kubectl` to your Convox created Kubernetes cluster by exporting a [kubeconfig](../../reference/cli/rack#rack-kubeconfig) that will connect you to a Kubernetes API Proxy running inside your Rack. This allows you to use `kubectl` without directly exposing the credentials for your Kubernetes cluster. For example if your Rack is called `myrack` you could point your local `kubectl` to your Rack cluster as follows
+
+```sh
+$ convox switch myrack
+$ convox rack kubeconfig > /tmp/myrack-config
+$ export KUBECONFIG=/tmp/myrack-config
```

Comment:
For a real example we should probably have them put this file in `~/.kube` so it doesn't get autodeleted.